sdk/storage/Azure.Storage.Queues/README.md

```diff
@@ -86,7 +86,7 @@ queue.SendMessage("third");
 foreach (QueueMessage message in queue.ReceiveMessages(maxMessages: 10).Value)
 {
     // "Process" the message
-    Console.WriteLine($"Message: {message.MessageText}");
+    Console.WriteLine($"Message: {message.Body}");
```

Comment:
This deserves an update in the CHANGELOG too with a few before/after code snippets to make it painless for anyone migrating.

Reply:
added. please let me know if it's verbose enough.